ibis/backends/duckdb/tests/test_udf.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import annotations
```

Comment:
This file should likely be moved to the backend test suite given how similar it is between the backends that support Python UDFs.

Reply:
Done!